frontend/src/themes/typography.ts

```diff
@@ -1,3 +1,5 @@
+import { getByPlaceholderText } from "@testing-library/react";
```

Comment:
what is this used for?

Reply:
I'm pretty sure I didn't add that...I can remove it if you would like

Reply:
I have removed it